theseus/geometry/se2.py

```diff
@@ -136,7 +136,7 @@ def theta(self, jacobians: Optional[List[torch.Tensor]] = None) -> torch.Tensor:
     def translation(self) -> Point2:
         return Point2(tensor=self[:, :2])
 
-    def xy(self, jacobians: Optional[List[torch.Tensor]] = None) -> torch.Tensor:
+    def xy(self, jacobians: Optional[List[torch.Tensor]] = None) -> Point2:
```

Comment:
I guess now it is same as translation but also gives jacobian. Maybe we add jacobian to translation and alias xy (or vice-versa)?

Reply:
Good catch. I'll refactor a bit.

Reply:
I made `translation` an alias for `xy()`.